src/components/SearchBar/SearchBar.types.ts

```diff
@@ -0,0 +1,21 @@
+import type { TextFieldProps } from '@mui/material';
+
+export interface SearchBarProps
```

Comment:
CDD guidelines require types/props to use a `Props` prefix. The interface is named SearchBarProps rather than starting with 'Props'. Please follow the project's naming convention for prop types.

**Sugerencia:**
```typescript
Rename the interface to follow the prefix convention, for example: export interface PropsSearchBar extends Omit<...> { ... } and update imports.
```